README.md

```diff
@@ -3,3 +3,10 @@
 A next generation search engine for researchers that supports searching with LaTeX math script.
 
 Refer to our [wiki](https://github.com/CornellDataScience/MathSearch/wiki) to get started :-)
+
+
+### Contribution List:
+2022 Fall Leads: Alexander Wang, Kaitlyn Chen  
+2023 Spring Leads: Emerald Liu, Johann Lee  
+2023 Fall Leads: Travis Zhang, Pun Chaixanien  
```

Comment:
Put Tech Lead and PM

Reply:
I decided not to do that because we wouldn't have the same structure for 2022 fall and 23 Spring
Do you think it's ok to just summarize as leads?